.gitignore

```diff
@@ -308,6 +308,7 @@ vignettes/*.pdf
 # pkgdown site
 docs/*
 !docs/README.md
+!docs/gtfs
```

Comment:
Quarto rendering will create an .html file, do we want to ignore it since the .md file is already there?

Reply:
Quarto does weird things with .mds. I discovered if you use an uppercase file extension `.MD`, quarto thinks it's an intermediate file in the quarto render process and then deletes it. Don't need the html file, but bit confused on why we're using quarto here.